imports/api/HistogramDB.js

```diff
@@ -1,11 +1,16 @@
+// @flow
+
 import { Meteor } from 'meteor/meteor';
 import { Mongo } from 'meteor/mongo';
+import type { HistogramState } from '../shared/state.models';
 
-export const HistogramDB = new Mongo.Collection('histogramdb');
+export const HistogramDB: HistogramState = new Mongo.Collection('histogramdb');
```

Comment:
I understand that it's not simply an object literal. However, the point behind typing is to prevent logic errors. As such, we could type this as a union between a `Mongo.Collection` and some underlying type that describes the collection. This will help us keep track of what is actually kept in the state, and prevent bugs from creeping in.